IPLD-Path.md

```diff
@@ -0,0 +1,20 @@
+# [WIP] IPLD Path v1
+
+An IPLD Path is a string identifier used for deep references into IPLD
+graphs.
+
+IPLD Path's are constructed following the same constraints as [URI Paths](https://tools.ietf.org/html/rfc3986#section-3.3).
```

Comment:
Do we also percent encode for escaping slashes?

Reply:
Yes.